platform/ios/src/MGLMapView.mm

```diff
@@ -1658,23 +1658,6 @@ - (void)handleSingleTapGesture:(UITapGestureRecognizer *)singleTap
         }
     }
     
-    // Handle the case of an offset annotation view by converting the tap point to be the geo location
-    // of the annotation itself that the view represents
-    for (MGLAnnotationView *view in self.annotationContainerView.annotationViews)
```

Comment:
This loop has been removed and the offset logic moved deeper into the annotation query.
- tapPoint is mutated within this loop, potentially leading to drifting away from the original tap
- With an offset, the tap point can end up outside the annotation view (but still within the bounds of the source annotation) - this leads to annotations being selected when it looks like they shouldn't.

Reply:
I'd like to understand what the point of this loop was, in case I'm missing something obvious 🤔